src/components/select-field/BaseSelectField.js

```diff
@@ -205,7 +206,17 @@ class BaseSelectField extends React.Component<Props, State> {
                     this.closeDropdown();
                 }
                 break;
-            // no default
+            default: {
+                stopDefaultEvent(event);
+                const lowerCaseKey = key.toLowerCase();
+                const optionIndex = options.findIndex(
```

Comment:
IE doesn't support `array.findIndex`. Can we use the lodash `_.findIndex` method instead?